plugins/vacation-calendar/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@axis-backstage/plugin-vacation-calendar",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^1.5.0",
+    "@backstage/core-components": "^0.14.8",
+    "@backstage/core-plugin-api": "^1.9.3",
+    "@backstage/errors": "^1.2.4",
+    "@backstage/plugin-catalog-react": "^1.12.1",
+    "@backstage/theme": "^0.5.6",
+    "@date-io/luxon": "3.0.0",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/pickers": "^3.3.11",
+    "@microsoft/microsoft-graph-types": "^2.40.0",
+    "@mui/lab": "^5.0.0-alpha.170",
+    "@mui/material": "^5.15.20",
+    "@mui/x-date-pickers": "^7.7.0",
+    "@tanstack/react-query": "4.29.1",
```

Comment:
This is really old, but we can update it later.

Reply:
How do you mean?

Reply:
Sorry, meant "@tanstack/react-query"!